packages/workspace/src/migrations/update-13-0-0/config-locations/config-locations.spec.ts

```diff
@@ -91,7 +90,6 @@ describe('update to v13 config locations', () => {
       });
       expect(workspaceJson.projects.a.tags).toEqual(['test']);
       expect(workspaceJson.cli).not.toBeDefined();
-      expect(workspaceJson.defaultProject).not.toBeDefined();
```

Comment:
Again, I think if they have it, we should move it over.

Reply:
Migration test